### PR TITLE
Initial upload of roadmap for 2.7

### PR DESCRIPTION
**Pull request by thaumos**
##### SUMMARY

Roadmap doc for 2.7

##### ISSUE TYPE

 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by jctanner** [comment]
missing native types

**Comment by thaumos** [comment]
@jctanner added in native types.

**Review by jctanner — APPROVED** [review]

**Comment by gundalow** [comment]
Fixed the CI issues, who else needs to review before we can get this live?

**Review by gundalow — APPROVED** [review]

File: docs/docsite/rst/roadmap/ROADMAP_2_7.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+* Refactor ``netconf_config`` module to add additional functionality. `#proposal#104 <https://github.com/ansible/proposals/issues/104>`_
+
+Windows
+-------
+
+General
+=======
+
+* Investigate the cause of WinRM HTTPS read timeouts
+* WinRM persistence
```
Comment (nitzmahone) [inline docs/docsite/rst/roadmap/ROADMAP_2_7.rst]: WinRM connection persistence (improved performance)

Comment (nitzmahone) [inline docs/docsite/rst/roadmap/ROADMAP_2_7.rst:96]: WinRM connection persistence (improves performance)

**Review by nitzmahone — CHANGES_REQUESTED** [review]
minor clarification on WinRM persistence, but otherwise LGTM

**Pull request merged**